battenberg/cli.py

```diff
@@ -11,6 +11,7 @@
 sys.path.append(os.path.join(os.path.dirname(__file__), '..'))  # noqa: E402
 
 
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
```

Comment:
Logging convention is to log all output to stderr unless the output is the purpose of the application e.g. grep results go to stdout but logs of where grep is searching would go to stderr.

Reply:
I think I was reviewing at the same time you merged :)

Reply:
Thanks for chasing me up on this, I remember having this conversation with you in the past now 😅 My copy/paste fail there, surprisingly a lot of the top results for logging setup do this 😞 

https://github.com/zillow/battenberg/pull/17